modules/vm/outputs.tf

```diff
@@ -0,0 +1,4 @@
+output "name" {
+  value       = azurerm_linux_virtual_machine.server.name
+  description = "Name of the created VM"
+}
```

Comment:
O sa avem nevoie de ip-ul de la vms in output ca sa ne putem conecta prin ssh si ca sa avem mai vizibil in consola ce ip-uri sunt folosite. Ele o sa se schimbe le fiecare deploy